corehq/apps/es/client.py

```diff
@@ -365,14 +365,23 @@ def _validate_single_index(index):
         elif "*" in index:
             raise ValueError(f"refusing to operate with index wildcards: {index}")
 
-    def reindex(self, source, dest, wait_for_completion=False, refresh=False):
+    def reindex(self, source, dest, wait_for_completion=False,
+                refresh=False, batch_size=1000, purge_ids=False):
```

Comment:
This module and method is well covered by tests. Will you consider covering these new kwargs as well? Good test coverage is valuable asset in regards to the future upgrade.

Reply:
For clarity, this review is not blocking.

Reply:
> For clarity, this review is not blocking.

Thanks for the clarification. I do plan to add those and request review again before merging, I have just been caught up in other things.